README.adoc

```diff
@@ -1,3 +1,42 @@
+= C++ example project scanned on SonarCloud using Travis CI
+// URIs:
+:uri-qg-status: https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_macos-cmake-travis-sc
+:img-qg-status: https://sonarcloud.io/api/project_badges/measure?project=sonarsource-cfamily-examples_macos-cmake-travis-sc&metric=alert_status
+:uri-build-status: https://app.travis-ci.com/sonarsource-cfamily-examples/macos-cmake-travis-sc
+:img-build-status: https://app.travis-ci.com/sonarsource-cfamily-examples/macos-cmake-travis-sc.svg?branch=main
+
+image:{img-build-status}[Build Status, link={uri-build-status}]
+image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]
+
+*This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_macos-cmake-travis-sc[SonarCloud]!*
+
+:note-caption: :information_source:
+NOTE: This simple example ignores https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-cache[analysis cache] and https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#parallel-code-scan[multithreaded execution] both of which can boost analysis significantly.
+
+It is very easy to analyze a C, C++ and Objective-C project with SonarCloud https://docs.travis-ci.com/user/sonarcloud/[on Travis CI]:
+
+. Create a `sonar-project.properties` file to store your configuration
+. Store the value of your token as SONAR_TOKEN environment variable in your Repository Settings
```

Comment:
Not true in this example, is it? The encrypted SONAR_TOKEN is written inside the `.travis.yml`

Reply:
The README of linux-autotools-travis-sc doesn't have this line whereas the README of linux-cmake-travis-sc has it.
I think we can remove it in both READMEs

Reply:
Indeed, well spotted

Reply:
removed.